lib/StatusIndicator/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const StatusIndicator = props => 
+  <span>
+    <i className="fa fa-circle" style={{ color: props.color }}></i> { props.label }
+    { props.children }
+  </span>
+
+StatusIndicator.propTypes = {
+  color: PropTypes.string,
+  label: PropTypes.string,
```

Comment:
we weren't sure what to add here, as it might not always be a string, and it isn't required. Do you have any suggestions @kyleharper ?